_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Prometheus SQL and Metrics Analytics Observability plugin
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Prometheus SQL and Metrics Analytics Observability plugin
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can use two new additional features within the **Observability** application in OpenSearch Dashboards: the Metrics UI for visualization, and connecting to Prometheus to gather metrics.
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
+
+## Metrics Analytics UI
+
+The Metrics Analytics UI is a new component in the **Observability** application for OpenSearch Dashboards, as seen in the following image.
+
+![Metrics UI example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics1.png)
+
+You can view visualizations from created metrics in the **Metrics analytics** section of the **Observability** application as seen in the following image.
+
+![Metrics UI example 2]({{site.url}}{{site.baseurl}}/images/metrics/metrics2.png)
+
+Additionally, you can save a visualization created with [Event analytics]({{site.url}}{{site.baseurl}}/observing-your-data/event-analytics/) as seen in the following image.
+
+Only queries that include a Time Series visualization and stats/span can be saved as a metric.
+{: .note }
+
+![Metrics UI example 3]({{site.url}}{{site.baseurl}}/images/metrics/metrics3.png)
+
+## Prometheus connection
+
+You can create a connection from [Prometheus](https://prometheus.io/) to OpenSearch via the SQL plugin in order to view metrics collected from Prometheus in OpenSearch Dashboards.
+
+![Prometheus example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics4.png)
+
+Upon selection of a metric, the **Metrics explorer** will create a visualization for the selected metric to be displayed, as seen in the following image.
+
+![Prometheus example 2]({{site.url}}{{site.baseurl}}/images/metrics/metrics5.png)
+
+### Configuring a Prometheus data source connection to OpenSearch
+
+In order to configure a connection to Prometheus for metrics in OpenSearch Dashboards, you will need to create a file on your OpenSearch nodes named `datasources.json` containing the Prometheus data source settings. The following examples demonstrate the various Prometheus data source configurations using different authentication methods:
+
+- No authentication:
```

Comment:
I don't think we need bullet points here - it should flow by using normal paragraphs